src/ui/IngredientForm.tsx

```diff
@@ -0,0 +1,24 @@
+import { FieldArray } from 'formik';
+import Ingredient from './Ingredient';
+import { FC } from 'react';
+
+const IngredientForm: FC = () => {
+  return (
+    <FieldArray
+      name="ingredients"
+      render={(arrayHelpers) => {
+        return arrayHelpers.form.values.ingredients.map((_, index) => {
+          return (
```

Comment:
you can use an implicit return here by wrapping your return object in `()` i.e. `.map((_, index) => ({ ... })`

Reply:
although don't go crazy with implicit returns, because it can be nice to be able to quickly add code later. for ex, i never use implicit returns when defining components

e.g.
```
const IngredientForm: FC = () => {
  // because you want to be able to quickly add code here
  return (...)
}
```